### DISCUSSION TITLE
cloud type?

### DISCUSSION THREAD

**Discussion by qfmp1012** [niels747/2D-Weather-Sandbox#92] [General]
![Screenshot (14)](https://github.com/niels747/2D-Weather-Sandbox/assets/135496047/5a8374c5-0a27-49a3-9649-baa3d1bb5aa2)
what type of cloud is this? I have seen it appear lots of times above cold fronts and below larger clouds.

**Comment by GRolLevente** [comment]
It is possible and similar to this cloud: altocumulus castellanus.
